ipaserver/plugins/trust.py

```diff
@@ -466,10 +445,12 @@ def fetch_trusted_domains_over_dbus(myapi, forest_name):
             logger.error('Standard output from the helper:\n%s---\n', _stdout)
             logger.error('Error output from the helper:\n%s--\n', _stderr)
         raise errors.ServerCommandError(server=myapi.env.host,
-                                        error=_('Fetching domains from trusted forest failed. '
-                                                'See details in the error_log'))
+                                        error=_('Fetching domains from trusted'
+                                                ' forest failed. See details '
+                                                'in the error_log'))
```

Comment:
Either add the space to end of each line or beginning of each line.

You can also safe a lot of *space* with a different indention style:

```
raise errors.ServerCommandError(
    server=myapi.env.host,
    error=_('Fetching domains from trusted...'
)
```